rancher2/schema_machine_config_v2_vsphere.go

```diff
@@ -206,6 +206,11 @@ func machineConfigV2VmwarevsphereFields() map[string]*schema.Schema {
 			Default:     "443",
 			Description: "vSphere Port for vCenter",
 		},
+		"graceful_shutdown_timeout": {
```

Comment:
Is this field not also Computed? Does only the Global one need to be?

Reply:
No, it is not. It must be set explicitly. 
Which global one do you refer to?